front/src/modules/trainschedule/components/Timetable/TimetableDepartureDate.tsx

```diff
@@ -0,0 +1,9 @@
+type TimetableDepartureDateProps = {
+  departureDate: string | null;
+};
+
+const TimetableDepartureDate = ({ departureDate }: TimetableDepartureDateProps) => (
+  <div className="scenario-timetable-departure-date">{departureDate}</div>
```

Comment:
Optional question: is it really worth introducing a new component for this simple div?

Reply:
you're absolutely right, creating a new component is pointless.